### ISSUE TITLE
Be functional equivalent to Karma on coverage reporting

### ISSUE THREAD

**Issue by ArnaudBuchholz** [ArnaudBuchholz/ui5-test-runner#85]
**Is your feature request related to a problem? Please describe.**
Karma is capable of reporting coverage for **all** sources of the project even if they are not **evaluated** during the tests.
See the karma configuration settings :
```js
    preprocessors: {
      'webapp/{,!(test)}/*.js': ['coverage']
    },
    coverageReporter: {
      includeAllSources: true,
```

When using *remote* or *legacy* modes, **only** the evaluated files are reported on.
It would be interesting to improve the process to also report on missing files to get a complete code coverage report.

**Describe the solution you'd like**
For *remote* mode, this can work only if we have access to source files (already tested).

**Describe alternatives you've considered**
Serving the files locally with `ui5-test-runner` but nyc does not include files which were not evaluated during the tests.

**Comment by ArnaudBuchholz** [comment]
Idea : inject a `.json` file enumerating all files matching the selection criteria but with no coverage information. Once merged, this may generate the report accordingly.

**Comment by ArnaudBuchholz** [comment]
- [ ] Condition the following based on `nyc.json`'s `all` setting
- [ ] Learn how to capture the instrumented information (`function cov_1zagmrg10g()`)
- [ ] [LEGACY] Scan the instrumented, no need to filter, and build an `initial.json` coverage structure containing coverage information for all files
- [ ] Learn how to leverage `nyc.json` includes and excludes
- [ ] [REMOTE] If possible, scan the URL, filter based on `nyc.json`, and build an `initial.json` coverage structure containing coverage information for all files
- [ ] Merge extracted coverage information with `initial.json`

**Comment by ArnaudBuchholz** [comment]
Fixed in `5.0.0`
